SudoScript.Core/Parser.cs

```diff
@@ -193,7 +193,7 @@ private static ParameterCellNode ParseParameterCell(TokenStream stream)
     private static FunctionCallNode ParseFunctionCall(TokenStream stream) 
     {
         List<ArgumentNode> arguments = new List<ArgumentNode>();
-        Token unionToken = new Token(TokenType.Identifier, "Union", "", 0, 0, "");
+        Token unionToken = new Token(TokenType.Identifier, "Union", 0, 0, "");
```

Comment:
This should also include some row and collumn definition. Maybe try getting it from the cell token or something.